controllers/api.js

```diff
@@ -312,7 +318,7 @@ router.post('/payinvoice', async function (req, res) {
       let inv = {
         payment_request: req.body.invoice,
         amt: info.num_satoshis, // amt is used only for 'tip' invoices
-        fee_limit: { fixed: Math.floor(info.num_satoshis * 0.005) + 1 },
+        fee_limit: { fixed: Math.floor(info.num_satoshis * internalFee) + 1 }, // fee setting was 0.005 now set as internalFee
```

Comment:
`internalFee` is now set to `0.003` instead of `0.005`. Is this intentional?

Reply:
I do belive this is a typo that has been lingering ? makes no sense have Paym.fee at 0.03 and then specify a *tip* fee at 0.05 ?